src/components/data/data-utils.js

```diff
@@ -1057,12 +1075,46 @@ export const MultipleEntityEditor = ({ entityType, entityNames, attributeNames,
           ])
         ]),
         attributeToEditTouched ? h(Fragment, [
-          p({ style: { fontWeight: 'bold' } }, ['Change selected values to:']),
-          h(AttributeInput, {
-            value: newValue,
-            onChange: setNewValue,
-            entityTypes
-          }),
+          div({ style: { display: 'flex', flexDirection: 'column', marginBottom: '1rem' } }, [
+            h(IdContainer, [id => h(Fragment, [
+              label({ htmlFor: id, style: { marginBottom: '0.5rem', fontWeight: 'bold' } }, 'Operation'),
+              h(Select, {
+                id,
+                options: [
+                  { label: 'Set value', value: 'setValue' },
+                  { label: 'Convert type', value: 'convertType' }
+                ],
+                value: operation,
+                onChange: ({ value: newOperation }) => {
+                  setOperation(newOperation)
+                  if (newOperation === 'setValue') {
+                    setNewValue('')
+                  }
+                  if (newOperation === 'convertType') {
+                    setNewType({ type: 'string' })
+                  }
```

Comment:
Nit: Is the vertical space above warranted vs. ?

```suggestion
                  newOperation === 'setValue' ? setNewValue('')
                  newOperation === 'convertType' ? setNewType({ type: 'string' })
```

Reply:
Personally, I think this is the clearest way to write code like this. I think the alternatives below obscure the idea that the purpose of the expression is the side effect of `setState`.

```js
newOperation === 'setValue' && setNewValue('')
```

ESLint complains about this because of the [no-unused-expressions](https://eslint.org/docs/latest/rules/no-unused-expressions) rule.
```js
newOperation === 'setValue' ? setNewValue('') : undefined
```

Reply:
Ah, sorry for the typo (`?` instead of `&&`) in the original suggestion.

The arguments you made for not making the change are very reasonable 👍🏼